sky/packages/sky/lib/src/fn3/input.dart

```diff
@@ -18,14 +21,16 @@ typedef void StringValueChanged(String value);
 // http://www.google.com/design/spec/components/text-fields.html#text-fields-single-line-text-field
 const EdgeDims _kTextfieldPadding = const EdgeDims.symmetric(vertical: 8.0);
 
-class Input extends StatefulComponent {
+class Input extends Scrollable {
   Input({
     GlobalKey key,
     this.initialValue: '',
     this.placeholder,
     this.onChanged,
     this.keyboardType: KeyboardType.TEXT
-  }): super(key: key);
+  }): super(key: key,
```

Comment:
put a newline after the (, then indent two spaces for the arguments, then ) on a line aligned with the }.